cypress/e2e/tests/rbac/disable_keycloak.test.ts

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import * as data from "../../../utils/data_utils";
+import { getRandomApplicationData, login, patchTackleCR } from "../../../utils/utils";
+import { Stakeholders } from "../../models/migration/controls/stakeholders";
+import { Application } from "../../models/migration/applicationinventory/application";
+import { Assessment } from "../../models/migration/applicationinventory/assessment";
+let application = new Assessment(getRandomApplicationData());
+let stakeholder = new Stakeholders(data.getEmail(), data.getFullName());
+
+describe(["@tier1"], "Perform certain operations after disabling Keycloak", function () {
+    // Automates Polarion MTA-293
+    before("Disable Keycloak", function () {
+        login();
+        patchTackleCR("keycloak", false);
+
+        stakeholder.create();
+        application.create();
+    });
+
+    beforeEach("Load data", function () {
+        // RBAC rules for architect are applicable to admin as well
+        cy.fixture("rbac").then(function (rbacRules) {
+            this.rbacRules = rbacRules["architect"];
+        });
+    });
+
+    it("Auth disabled, Perform application assessment", function () {
+        application.perform_assessment("low", [stakeholder.name]);
+    });
+
+    it("Auth disabled, Verify presence of Review application button", function () {
+        Application.validateReviewButton(this.rbacRules);
+    });
+
+    after("Re-enable Keycloak", function () {
+        application.delete();
+        stakeholder.delete();
+        patchTackleCR("keycloak", true);
```

Comment:
Can we enable keycloak again before deleting the test data?

If the deletion of test data fails, we'd left keycloak disabled for the rest of the tests, and they may fail because of that

Reply:
@abrugaro , Although I've updated the PR, there are some issues with the mta-ui pod after re-enabling Keycloak because of which the clean up fails.
I'm currently investigating the issues. I'll switch the PR to RFR once the issues have been resolved.

> Can we enable keycloak again before deleting the test data?

Reply:
Tests fail on 6.2 because of this bug. This bug is targeted for MTA 7.0.

https://issues.redhat.com/browse/MTA-1152